src/Department.java

```diff
@@ -1,7 +1,55 @@
 package application;
 
+import java.util.Random;
+
 public class Department {
+	public String name="Test";
+	public int id=123;
+	public String country="Neverland";
+	public String city="Gotham";
+
+
+	public Department(String name, int id, String country, String city) {
+		this.name = name;
+		this.id = id;
+		this.country = country;
+		this.city = city;
+	}
+
```

Comment:
Missing defaut constructor. It should look like this:

~~~java
public Department() {
    name = "Test";
    int id = 123;
    country = "Neverland";
    city = "Gotham";
}
~~~